sources/light_client.move

```diff
@@ -72,28 +75,33 @@ public struct LightClient has key, store {
 
 
 // === Init function for module ====
-fun init(ctx: &mut TxContext) {
-    let p = mainnet_params();
-    // https://btcscan.org/block/00000000000003a5e28bef30ad31f1f9be706e91ae9dda54179a95c9f9cd9ad0
-    let raw_header = x"010000009d6f4e09d579c93015a83e9081fee83a5c8b1ba3c86516b61f0400000000000025399317bb5c7c4daefe8fe2c4dfac0cea7e4e85913cd667030377240cadfe93a4906b50087e051a84297df7";
-    let lc = new_light_client(p, 201600, vector[raw_header], ctx);
-    transfer::transfer(lc, tx_context::sender(ctx));
+fun init(_ctx: &mut TxContext) {
+    // let p = mainnet_params();
+    // // https://btcscan.org/block/00000000000003a5e28bef30ad31f1f9be706e91ae9dda54179a95c9f9cd9ad0
+    // let raw_header = x"010000009d6f4e09d579c93015a83e9081fee83a5c8b1ba3c86516b61f0400000000000025399317bb5c7c4daefe8fe2c4dfac0cea7e4e85913cd667030377240cadfe93a4906b50087e051a84297df7";
+    // let lc = new_light_client(p, 201600, vector[raw_header], ctx);
+    // transfer::transfer(lc, tx_context::sender(ctx));
```

Comment:
why commenting?

Reply:
Creating a light client object by calling function new_btc_light_cleint is better than this way.
Can I remove this?